package.json

```diff
@@ -83,7 +81,15 @@
     "redux": "^3.6.0",
     "redux-thunk": "^2.2.0",
     "reselect": "^2.5.4",
+    "sass-loader": "6.0.7",
```

Comment:
Calypso is using version `7.*.*` of `sass-loader`, but until [this regression](https://github.com/webpack-contrib/sass-loader/issues/556) is fixed we must stay in `6.*.*`.